Linked Update User Details Screen to Navigation Menu.

AananAbdullah opened the pull request.

AananAbdullah committed:
Linked Update User Details to Navigation.

Ninny659 commented on navigation/DrawerNavigator.tsx:
UpdateUserDetailsParamList needs an apostrophe after it, otherwise, the app won't open.

Ninny659 commented on screens/UpdateUserDetails.tsx:
How the system is handling handleUpdate and handleSave seems to be incorrect. I would advise rethinking the current approach. I have done a Firebase concept I would like to show you, as well as a few ways I have managed to modify buttons on the program to better suit the intended style, I will demonstrate these in our next meeting 25/4.

Ninny659 reviewed (changes requested):
I have left some comments on the appropriate dot tsx files that need changes. I would advise reviewing current methods in Firebase, as I believe these will not function accordingly. Furthermore, I have made some slight changes on my local machine; devising an approach that may work more accordingly with Firebase and its functions. I will demonstrate in the next meeting 25/4 to give a better idea. Thanks, Jordan.

AananAbdullah committed:
Merge branch 'Main' of https://github.com/Chameleon-company/Evoleon into aanan_update

AananAbdullah committed:
Modified Update User Details function

Modified update user details function in firebase.js and some style elements in UpdateUserDetails.tsx.
